pkg/api/types.go

```diff
@@ -2,8 +2,20 @@ package api
 
 import docker "github.com/fsouza/go-dockerclient"
 
+const (
+	DefaultNamespace    = "io.openshift.s2i."
+	KubernetesNamespace = "k8s.io/"
```

Comment:
thought @smarterclayton said k8s was using docker style labels for their images (not their annotations of course)?